apps/builder/src/features/editor/components/TypebotHeader/TypebotHeader.tsx

```diff
@@ -52,6 +52,39 @@ export const TypebotHeader = () => {
     setRightPanel(RightPanel.PREVIEW)
   }
 
+  const isUndoShortcut = (event: KeyboardEvent) => {
+    const key = event.key.toLowerCase()
+    const cmdHeld = event.metaKey
+    const ctrlHeld = event.ctrlKey
+    const Z = 'z'
+
+    return (cmdHeld && key === Z) || (ctrlHeld && key === Z)
+  }
```

Comment:
It would probably be better if we don't create variables each time a key is hit. So I would suggest:

```suggestion
  const isUndoShortcut = (event: KeyboardEvent) => (event.metaKey || event.ctrlKey) && event.key === 'z'
```

Reply:
refactored